burn-book/src/building-blocks/tensor.md

```diff
@@ -274,20 +277,20 @@ Those operations are only available for `Float` tensors.
 | `tensor.var_bias(dim)`                       | N/A                                |
 | `tensor.var_mean(dim)`                       | N/A                                |
 | `tensor.var_mean_bias(dim)`                  | N/A                                |
-| `tensor.zeros_like()`                        | `torch.zeros_like(tensor)`         |
 
 ### Int Operations
 
 Those operations are only available for `Int` tensors.
 
 | Burn API                                         | PyTorch Equivalent                                      |
-| ------------------------------------------------ | ------------------------------------------------------- |
+|--------------------------------------------------|---------------------------------------------------------|
 | `tensor.arange(5..10, device)`                   | `tensor.arange(start=5, end=10, device=device)`         |
 | `tensor.arange_step(5..10, 2, device)`           | `tensor.arange(start=5, end=10, step=2, device=device)` |
 | `tensor.float()`                                 | `tensor.to(torch.float)`                                |
 | `tensor.from_ints(ints)`                         | N/A                                                     |
 | `tensor.int_random(shape, distribution, device)` | N/A                                                     |
 | `tensor.cartesian_grid(shape, device)`           | N/A                                                     |
+| `tensor.one_hot_encode(num_classes)`             | N/A                                                     |
```

Comment:
Sure, changed the name. Yeah I also agree that the other `one_hot ` function is a bit limited. Just out of curiosity, we're not removing it because it is a breaking change?

Reply:
Yeah I think I'd deprecate it first and then remove it since it is user-facing.